src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+const WEEKLY_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  if (days >= 7) {
+    return totalCost - WEEKLY_DISCOUNT;
+  } else if (days >= 3) {
+    return totalCost - SHORT_TERM_DISCOUNT;
```

Comment:
```suggestion
    return totalCost - WEEKLY_DISCOUNT;
  }
  
  if (days >= 3) {
    return totalCost - SHORT_TERM_DISCOUNT;
```
`else` is redundant here as the code will not go further after `return` anyway